src/utils/index.js

```diff
@@ -2,14 +2,18 @@ import moment from 'moment';
 import qs from 'query-string';
 
 import history from '../data/history';
+import { DATE_FORMAT } from '../data/constants';
 
 const getDateString = date => (date ? moment.utc(date).format('YYYY-MM-DD') : '');
+const getTimeString = date => (date ? moment.utc(date).format('HH:mm') : '');
 
 const isValidDate = (dateStr) => {
   const date = moment(dateStr);
   return moment(dateStr) && date.isValid();
 };
 
+const concatDateTime = (dateStr, timeStr) => moment.utc(dateStr + timeStr, 'YYYY/MM/DD HH:mm').format(DATE_FORMAT);
```

Comment:
Added this comment in above, I think this method would make more sense if we used string interpolation rather than concatenation here as to me it looks like you're adding two dates together.